app/relationship.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class relationship extends Model
```

Comment:
The first letter of the class name should be capitalized based on Laravel convention. Please capitalize also the first letter of file name.

Reply:
I had modified.
I will write code based on the rules from now on.